docs/contributing/README.md

```diff
@@ -1,14 +1,39 @@
 # Developing the Provider
 
-If you wish to work on the provider, you'll first need [Go](http://www.golang.org) installed on your machine (see [Requirements](../../README.md)).
-The easiest way to do this is to use devbox and run `devbox shell`.
+Preferably use [Devbox](https://www.jetpack.io/devbox/docs/installing_devbox/) for dependencies and development. If you wish to install dependencies manually, see [Requirements](../../README.md).
 
-To compile the provider, run `make install`. This will build the provider and put the provider binary in the `$GOPATH/bin` directory.
+## Getting dependencies
 
-*Note:* if you're on macOS, change `OS_ARCH` in `Makefile` to `darwin_amd64` (Intel) or `darwin_arm64` (Apple silicon)
+```sh
+devbox install
+devbox shell
+```
+
+## Running provider locally
+You have 2 options to run the provider locally:
+
+1. Use development overrides (preferred):
+    1. copy `dev.tfrc.example` to `dev.tfrc`
+    1. update `dev.tfrc` with `echo $GOBIN`
```

Comment:
To follow up on https://linear.app/sleuth/issue/BROC-373/add-shortcut-as-incident-impact-source-in-tf#comment-c1ef61ac I'm now setting up Codespace for this repo.

And I'm already stuck at step 2 😊, because `echo $GOBIN` returns an empty line for me:

```sh
@zigomir ➜ /workspaces/terraform-provider-sleuth (main) $ echo $GOBIN


```

What else can I do here?

I also tried
```sh
@zigomir ➜ /workspaces/terraform-provider-sleuth (main) $ go env | grep GOBIN
GOBIN=''
```

maybe empty string is correct ok though, I'll try that (sorry for spam) :)

Reply:
yeah empty $GOBIN should be fine. i think it defaults to /usr/local/bin - are you in devbox or directly locally

Reply:
i'm actually in a codespace (don't wanna deal with go and terraform locally just yet), so you could take a look directly: https://refactored-cod-7v7rppwwv53rr7v.github.dev/

Reply:
I don't have a permissions it seems 🤔

Reply:
Ah I now see Codespaces are scoped to a single user. Oh well, let's tackle this at 15:00 then.